test/parallel/test-uv-errno.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const util = require('util');
+const uv = process.binding('uv');
+
+const keys = Object.keys(uv);
+
+keys.forEach((key) => {
+  if (key === 'errname')
+    return;
+
+  assert.doesNotThrow(() => {
```

Comment:
I think it mangles the actual error a little bit:
https://github.com/nodejs/node/blob/467385a49b659b704973b8195328775b620ae6ab/test/parallel/test-assert.js#L485-L497